tests/readme.md

```diff
@@ -3,30 +3,44 @@ Hypernode Magerun Addons - Tests
 
 Within this folder PHPUnit tests are included. 
 These tests are automatically ran by travis on push and pull request in various environment setups related to Hypernode.
+The test setup is strongly related the the one used by n98-magerun.
 
 # Configuring your local environment
 
 The local environment requires:
 
-* an environment variable pointing to a Magento setup
+* an environment variable pointing to a Magento setup or the .n98-magerun stopfile pointing to ./public (auto generated by test_setup.sh)
```

Comment:
Maybe make '.n98-magerun stopfile' link to a page explaining what that is (maybe [this one](http://magerun.net/released-n98-magerun-version-1-95-0/)). It is not immediately apparent that that's a conventional magerun concept instead of something specific relating to our testing configuration.